client/gutenberg/extensions/publicize/connection.jsx

```diff
@@ -12,14 +12,27 @@
  */
 import { Component } from '@wordpress/element';
 import { Disabled, FormToggle, Notice, ExternalLink } from '@wordpress/components';
+import { compose } from '@wordpress/compose';
 import { withSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 import PublicizeServiceIcon from './service-icon';
+import getSiteFragment from 'gutenberg/extensions/presets/jetpack/editor-shared/get-site-fragment';
 
+/**
+ * Return a link to the Sharing page, whether it's on Calypso or WP Admin.
+ *
+ * @returns {string} Link to Sharing page.
+ */
+const getSharingLink = () =>
```

Comment:
Note that we already have logic for this: https://github.com/Automattic/wp-calypso/blob/1f2f6e653dde5ca2e8abd18059273909a0d9b932/client/gutenberg/extensions/publicize/settings-button.jsx#L23

I'd expect we don't want to duplicate this and would want to reuse it throughout the extension.

Reply:
No longer needed since relevant code was removed.